### DISCUSSION TITLE
ppocr识别1和7难以区分

### DISCUSSION THREAD

**Discussion by omaiyiwa** [PaddlePaddle/PaddleOCR#14034] [Q&A]
我通过二值化图像,以及腐蚀膨胀等对数字进行特征增强,但是1和7总是难以区分,缩放不同的图片大小,不同的
核总是有问题,但是图中1和7看起来区别很大,有只识别数字的模型吗,或者有其它解决方案
![PixPin_2024-10-18_10-01-14](https://github.com/user-attachments/assets/67ea45e6-46be-46dd-b259-c5898dee3d37)

**Comment by GreatV** [comment]
从我们的模型列表里找找吧 https://paddlepaddle.github.io/PaddleOCR/latest/ppocr/model_list.html

**Reply by omaiyiwa** [reply]
之前都试过了,不太行

**Reply by GreatV** [reply]
自己训练一下吧
